source/includes/gcp/_externalips.md

```diff
@@ -0,0 +1,136 @@
+### External IPs
+External IP address to an instance or a forwarding rule if you need to communicate with the Internet, with resources in another network, or need to communicate with a resource outside of Compute Engine.
+
+<!-------------------- LIST EXTERNAL IPs -------------------->
+
+#### List external IPs
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/externalips"
+
+# The above command returns JSON structured like this:
+```
+```json
+{
+  "data": [
+    {
+      "address": "35.203.89.240",
```

Comment:
Are we missing a bracket here? these 2 objects are very different one from the other.